docs/source/help.rst

```diff
@@ -5,7 +5,7 @@
 Need Help?
 =================
 
-At some point of usage of the `PyExperimenter` you don't know how to continue?
+At some point of usage of the ``PyExperimenter`` you don't know how to continue?
```

Comment:
I would simplify this to just: Do you have further questions or a problem with ``PyExperimenter`` and don't know how to continue?